keentools_facebuilder/camera_input.py

```diff
@@ -0,0 +1,21 @@
+import numpy as np
+from pykeentools import FaceBuilderCameraInputI
+
+from . import FBCameraItem
+from .settings import get_main_settings, FBSceneSettings
+
+
+class FaceBuilderCameraInput(FaceBuilderCameraInputI):
+    @staticmethod
+    def _camera_at(frame) -> FBCameraItem:
+        settings: FBSceneSettings = get_main_settings()
+        return settings.get_camera(settings.current_headnum, frame - 1)
```

Comment:
What is the "frame" variable responsible for? Is this the camera number in settings or keyframe number? Anyway (frame - 1) looks suspicious. Camera keyframes (keyframe_id) can be in [1, 5, 6, 9] by ex. I think it needs a search camera by keyframe_id